modules/evm-manager/src/lib.rs

```diff
@@ -0,0 +1,250 @@
+// This file is part of Acala.
+
+// Copyright (C) 2020-2021 Acala Foundation.
+// SPDX-License-Identifier: GPL-3.0-or-later WITH Classpath-exception-2.0
+
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+//! # Evm Manager Module
+//!
+//! ## Overview
+//!
+//! Evm Manager module provide a two way mapping between CurrencyId and
+//! ERC20 address so user can use ERC20 address as LP token.
```

Comment:
If this module is only for currency id and erc20 address mapping, I think the name `evm-manager` is too general.

Reply:
Maybe will manage the factory contract.

Reply:
Then the docs could be changed like:

```
Evm manager module provides common support features for Evm, including:
- A two way mapping blabla...
```

And we can add more the the list when factory contract added in the future.